Tiltfile.dev

```diff
@@ -1,4 +1,5 @@
 custom_build(
+  # TODO: this command is to be revisited when we know more about the AWS setup
```

Comment:
I think we can remove this TODO as currently the latest image of lfp-pay-api is what we have in eu-west-2.

Reply:
Will remove.